src/Neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -147,7 +148,7 @@ internal protected UInt160 CreateMultisigAccount(int m, ECPoint[] pubKeys)
         /// The implementation of System.Contract.NativeOnPersist.
         /// Calls to the <see cref="NativeContract.OnPersistAsync"/> of all native contracts.
         /// </summary>
-        protected internal async void NativeOnPersistAsync()
+        protected internal async Task NativeOnPersistAsync()
```

Comment:
I'm not sure if need to use `ContractTask` in all of the renamed method related to native contracts

Reply:
Using `async void` could have the process lock up.

Reply:
I mean `ContractTask` instead of `Task` we have some `is ContractTask`

Reply:
@shargon you keep breaking my PRs.... lol 🥇 

What i was saying is it was `void` so `no` you don't need to change to `ContractTask`.

Reply:
Plus I tried `ContractTask` and it didn't work.